lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -197,6 +197,19 @@ const ItineraryGridSmall = styled.div`
   }
 `
 
+const BLUR_AMOUNT = 3
+const blurAnimation = keyframes`
+ 0% { filter: blur(${BLUR_AMOUNT + 1}px); }
+ 50% { filter: blur(${BLUR_AMOUNT}px) }
+`
+
+const LoadingBlurred = styled.span<{ loading: boolean }>`
+  ${(props) => props.loading && `filter: blur(${BLUR_AMOUNT}px)`};
+  animation-name: ${(props) => (props.loading ? blurAnimation : '')};
+  animation-duration: 1s;
+  animation-iteration-count: infinite;
```

Comment:
Is it possible to add a transition to the filter so that the blur sort of fades out instead of being instant? Otherwise I think it looks good. I don't really notice the animation that much, but that's fine.

Reply:
Good catch! Looks much better thanks!